recipes/cccl/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
```

Comment:
not required since is not a generic folder

Reply:
I've renamed it to a generic folder structure